packaging/rpm/SPECS/stackable-agent.spec

```diff
@@ -0,0 +1,62 @@
+%define __spec_install_post %{nil}
```

Comment:
there are two stackable-agent.spec files. Which one is correct or are they both used?

Reply:
Removed the incorrect one.